### PR TITLE
adding known downstreams for reconciler-test

### PR DESCRIPTION
**Pull request by n3wscott**

### CODE REVIEW

File: .github/workflows/knative-downstream.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Downstream
+
+on:
+  pull_request:
+    branches: [ 'master', 'release-*' ]
```
Comment (vaikas) [inline .github/workflows/knative-downstream.yaml:19]: for the up&coming master->main rename, want to add that here now?

**Review by vaikas — APPROVED** [review]

**Comment by vaikas** [comment]
/hold
if you want to add main branch now. unhold at will.

File: .github/workflows/knative-downstream.yaml (hunk 1 above)
Comment (n3wscott) [reply .github/workflows/knative-downstream.yaml:19]: it will all have to get updated, but this is based on github, not the downstreams

**Comment by n3wscott** [comment]
/unhold

let's tackle  main to master all at once

**Pull request merged**